features/extra/hooks.js

```diff
@@ -7,14 +7,7 @@ const isType = (obj, type) => {
   return Object.prototype.toString.call(obj) === "[object " + type + "]";
```

Comment:
This file needs to be removed from this PR, can be prettified separately if required

Reply:
Good catch, not sure why thats there. Updated